c/driver/sqlite/sqlite_test.cc

```diff
@@ -190,10 +190,10 @@ class SqliteStatementTest : public ::testing::Test,
   void SetUp() override { ASSERT_NO_FATAL_FAILURE(SetUpTest()); }
   void TearDown() override { ASSERT_NO_FATAL_FAILURE(TearDownTest()); }
 
-  void TestSqlIngestUInt64() {
-    std::vector<std::optional<uint64_t>> values = {std::nullopt, 0, INT64_MAX};
-    return TestSqlIngestType(NANOARROW_TYPE_UINT64, values);
-  }
+  void TestSqlIngestUInt8() { GTEST_SKIP() << "Not implemented"; }
+  void TestSqlIngestUInt16() { GTEST_SKIP() << "Not implemented"; }
+  void TestSqlIngestUInt32() { GTEST_SKIP() << "Not implemented"; }
+  void TestSqlIngestUInt64() { GTEST_SKIP() << "Not implemented"; }
```

Comment:
My understanding is that an int overflow can happen (and I assumed that's why it's also skipped in PostgreSQL) since BIGINT in PostgreSQL and INTEGER in SQLite support up to 2^63 - 1 but `uint64_t` can go up to 2^64 - 1.

Reply:
We can allow the type and just reject values if they would overflow, though.